.github/workflows/publish.yml

```diff
@@ -0,0 +1,58 @@
+---
+name: Publish content
+
+on:
+  pull_request: ~
+  push:
+    branches: [ main ]
+
+  # Allow job to be triggered manually.
+  workflow_dispatch:
+
+# Cancel in-progress jobs when pushing to the same branch.
+concurrency:
+  cancel-in-progress: true
+  group: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+
+  publish-webserver:
+
+    runs-on: "ubuntu-latest"
+
+    name: Upload content to web server
+    steps:
+
+    - name: Acquire sources
+      uses: actions/checkout@v4
+
+    - name: Install Rclone
+      run: sudo apt-get install --yes --no-install-recommends --no-install-suggests rclone
+
+    - name: Configure Rclone
+      run: |
+        mkdir -p ~/.config/rclone
+        cat << EOF > ~/.config/rclone/rclone.conf
+
+        [datasets-webdav]
+        type = webdav
+        url = https://cdn.crate.io/downloads/datasets
+        vendor = other
+
+        EOF
+
+    # https://github.com/tailscale/github-action
+    - name: Connect using Tailscale
+      uses: tailscale/github-action@v3
+      with:
+        oauth-client-id: ${{ secrets.TS_OAUTH_CLIENT_ID }}
+        oauth-secret: ${{ secrets.TS_OAUTH_SECRET }}
+        tags: tag:ci
```

Comment:
Because using SSH is not an option, because it is blocked from outside, I am wondering if using Tailscale is the optimal option.

- https://github.com/tailscale/github-action
- https://tailscale.com/kb/1215/oauth-clients

Reply:
We did it like this now. Thanks a stack for your support, @WalBeh.